modules/installation-and-upgrade/pages/containerized-proxy-deployment.adoc

```diff
@@ -0,0 +1,75 @@
+[[containerized-proxy-deployment]]
+= Containerized proxy deployment
+
+It is possible to deploy containerized images in an environment without an internet connection.
+In such case, the images are copied from {suse} registry to an internal registry, or saved to a [literal]``tar`` file.
+
+You should first register Salt minion to {productname} server, and then re-use the same FQDN in a generation.
+
+
+
+[[image.from.suse.to.internal.registry]]
+== Image copying from {suse} registry to internal registry 
+
+
+This example illustrates deployment of {salt} proxies only.
+
+.Procedure: Deploying {salt} Proxy from containerized image
+[role=procedure]
+
+. Install [literal]``skopeo`` on a machine with access to [literal]``registry.suse.com``:
++
+----
+zypper in skopeo (can be {productname} server)
+----
++
+. Copy images between registries:
++
+----
+for image in httpd proxy-salt-broker squid ssh tftpd; do
+    skopeo copy --dest-tls-verify=false docker://registry.suse.com/suse/manager/4.3/proxy-$image:latest docker://(your_server)/proxy-$image
+done
+----
++
+. If the registry is unsecured, i.e. not configured with SSL, add the regitry domain to the section [literal]``insecure list`` on the containerized proxy virtual machine by editing:
++
+----
+/etc/containers/registries.conf
+----
++
+[NOTE]
+====
+If you want to copy the images to a registry with SSL, remove the property [literal]`--dest-tls-verify=false`.
```

Comment:
Shouldn't we invert the logic and have a note to add that parameter if the registry is not secured?

Reply:
@cbosdo - Would this work? "If the registry is not secured, add `--dest-tls-verify=false` ?

Reply:
yes it would